concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-9	9	ant (red)			4
+Marrison-1967-909-10	10	ant (white), termite	883	TERMITE	4
+Marrison-1967-909-11	11	arm	1673	ARM	2
+Marrison-1967-909-12	12	armlet			
+Marrison-1967-909-13	13	arrive	1387	ARRIVE	2
+Marrison-1967-909-14	14	arrow	977	ARROW	2
+Marrison-1967-909-15	15	ascend	705	GO UP (ASCEND)	2
+Marrison-1967-909-16	16	ashes	646	ASH	2
+Marrison-1967-909-17	17	ask (a question)	1309	ASK (INQUIRE)	4
+Marrison-1967-909-18	18	ask for	1828	ASK (REQUEST)	2
+Marrison-1967-909-19	19	at	1461	AT	2
+Marrison-1967-909-20	20	aunt	1272	AUNT	2
+Marrison-1967-909-21	21	awake / awaken	1602	WAKE UP	4
+Marrison-1967-909-22	22	axe	677	AXE	2
+Marrison-1967-909-23	23	baby	1289	BABY	2
+Marrison-1967-909-24	24	back (body)	1291	BACK	4
+Marrison-1967-909-25	25	bad	1292	BAD	2
+Marrison-1967-909-26	26	bag	2468	BAG	2
+Marrison-1967-909-27	27	bamboo	1927	BAMBOO	2
+Marrison-1967-909-28	28	banana	868	BANANA	2
+Marrison-1967-909-29	29	bank (edge)	113	SHORE	4
+Marrison-1967-909-30	30	bark (of tree)	1204	BARK	2
+Marrison-1967-909-31	31	basket (back, man's)			4
+Marrison-1967-909-32	32	basket (back, woman's)			4
+Marrison-1967-909-33	33	basket (generic)	1539	BASKET	4
+Marrison-1967-909-34	34	basket (large, storage)			4
+Marrison-1967-909-35	35	bat (animal)	1793	BAT	4
+Marrison-1967-909-36	36	be	1579	BE	2
+Marrison-1967-909-37	37	bead	1860	BEAD	2
+Marrison-1967-909-38	38	beam	1132	BEAM	2
+Marrison-1967-909-39	39	bean	832	BEAN	2
+Marrison-1967-909-40	40	bear (animal)	1820	BEAR	4
+Marrison-1967-909-41	41	bear (fruit)			4
+Marrison-1967-909-42	42	bear fruit	3054	BEAR FRUIT	2
+Marrison-1967-909-43	43	beard	678	BEARD	2
+Marrison-1967-909-44	44	beat	1665	BEAT	2
+Marrison-1967-909-45	45	beautiful	1427	BEAUTIFUL	2
+Marrison-1967-909-46	46	because	1157	BECAUSE	2
+Marrison-1967-909-47	47	become	1990	BECOME	2
+Marrison-1967-909-48	48	bed	1663	BED	2
+Marrison-1967-909-49	49	bee	665	BEE	2
+Marrison-1967-909-50	50	beetle	2242	BEETLE	2
+Marrison-1967-909-51	51	before	1882	BEFORE	2
+Marrison-1967-909-52	52	before; at; to	1461	AT	4
+Marrison-1967-909-53	53	begin	1806	BEGIN	2
+Marrison-1967-909-54	54	believe	1890	BELIEVE	2
+Marrison-1967-909-55	55	bellows			
+Marrison-1967-909-56	56	belly	1251	BELLY	2
+Marrison-1967-909-57	57	belows			
+Marrison-1967-909-59	59	belt (girdle)	772	BELT	4
+Marrison-1967-909-60	60	betel (leaf)			
+Marrison-1967-909-61	61	betel (nut)			
+Marrison-1967-909-62	62	between	689	BETWEEN	2
+Marrison-1967-909-63	63	big	1202	BIG	2
+Marrison-1967-909-64	64	bind	1917	TIE	2
+Marrison-1967-909-65	65	bird	937	BIRD	2
+Marrison-1967-909-66	66	birth, give	1195	GIVE BIRTH	2
+Marrison-1967-909-67	67	bite	1403	BITE	2
+Marrison-1967-909-68	68	bitter	887	BITTER	2
+Marrison-1967-909-69	69	black	163	BLACK	2
+Marrison-1967-909-70	70	bladder	1746	BLADDER	2
+Marrison-1967-909-71	71	bless	391	BLESS	2
+Marrison-1967-909-72	72	blind	1653	BLIND	2
+Marrison-1967-909-73	73	blood	946	BLOOD	2
+Marrison-1967-909-74	74	blow	176	BLOW (WITH MOUTH)	2
```

Comment:
this is unclear: could be blow (with mouth) or blow (of wind). So either leave it unlinked, or make a generic:

BLOW (WITH MOUTH OR OF WIND)

with the relations. This is a bad concept set then, but we often find that lists don't specify...

Reply:
I will leave it unlink for now.